modules/editor-one/assets/scss/_common.scss

```diff
@@ -0,0 +1,266 @@
+@import "../../../../assets/dev/scss/helpers/variables";
+@import "../../../../assets/dev/scss/helpers/mixins";
+@import "variables";
+@import "../../../../assets/dev/scss/direction/theme-light";
+
+@import "theme-light";
+
+@media (prefers-color-scheme: dark) {
+
+}
+
+#wpcontent,
+#wpbody-content,
+#wpbody-content > * {
+	background-color: var(--e-one-palette-background-default) !important;
```

Comment:
@heinvv what do you think of this approach? This should be loaded only on pages related to elementor of course, but it looks a little...agressive. Do you have a better approach to suggeste? Maybe should we target something more specifical?